tiltak/src/test/kotlin/no/nav/amt/tiltak/deltaker/repositories/NavAnsattRepositoryTest.kt

```diff
@@ -52,15 +53,18 @@ class NavAnsattRepositoryTest : FunSpec({
 	}
 
 	test("Update nav-ansatt så hent bør returnere oppdatert nav-ansatt") {
-		repository.upsert(upsertCmd)
-		val storedDbo = repository.getNavAnsattWithIdent(upsertCmd.navIdent)
-
-		repository.upsert(upsertCmd.copy(epost = null, telefonnummer = null))
-		val updatedDbo = repository.getNavAnsattWithIdent(upsertCmd.navIdent)
-
-		storedDbo!!.id shouldBe updatedDbo!!.id
-		updatedDbo.epost shouldBe null
-		updatedDbo.telefonnummer shouldBe null
+		repository.upsert(UpsertNavAnsattCommand(
+			navIdent = NAV_ANSATT_1.nav_ident,
+			navn = "Nytt navn",
```

Comment:
Flytte ut i variabler siden strengene brukes flere ganger, så testen blir mer lesbar

Reply:
Litt uening, strengene er brukt 2 steder i samme funksjon rett ved siden av hverandre, det er lettere å lese rå strenger enn å sjekke at det er samme variabel som blir brukt for å opprette og asserte. Men lesbarhet er en subjektiv ting.